### DISCUSSION TITLE
Mongolian Language Leader

### DISCUSSION THREAD

**Discussion by synesthesiam** [OHF-Voice/intents#375] [Language Leaders]
@chuk-a are you still interested in being a language leader for Mongolian?

**Comment by chuk-a** [comment]
yes, sir. I'd love to be a leader.

**Reply by synesthesiam** [reply]
Great! I'm adding some empty Mongolian (`mn`) templates to the repo. Please submit at least two small PRs with translations, and I'll get you added as language leader :+1:

**Reply by chuk-a** [reply]
Check my translations. Thank you.

**Reply by synesthesiam** [reply]
Very close. See my comments in the PR.

**Reply by chuk-a** [reply]
check my work pls

**Comment by synesthesiam** [comment]
Thanks, @chuk-a! Adding you as language leader :+1:
